app/models/organisation/organisation_type_concern.rb

```diff
@@ -74,6 +74,11 @@ def active_child_organisations_excluding_sub_organisations_grouped_by_type
       active_child_organisations_excluding_sub_organisations.group_by(&:organisation_type).sort_by { |type, department| type.listing_position }
   end
 
+  def active_child_organisations_excluding_sub_organisations_and_courts_and_tribunals
```

Comment:
This is getting pretty unwieldy. Would it be better to name this for what it does rather than what it doesn't do? Perhaps `supporting_bodies`?